### ISSUE TITLE
Katex: Can we make the trust options editable via config

### ISSUE THREAD

**Issue by fredericcogny** [shd101wyy/markdown-preview-enhanced#1498]
The [math rendering doc](https://shd101wyy.github.io/markdown-preview-enhanced/#/math) sends a link to the [KaTeX supported functions/symbols.](https://katex.org/docs/supported.html).

Among those the [html support](https://katex.org/docs/supported.html#html) states that

![image](https://user-images.githubusercontent.com/15909740/100639606-b2253c00-3335-11eb-98b3-40e6929c5cc2.png)

My request is therefore to make the [the trust option](https://katex.org/docs/options.html) editable via the `settings.json` if possible so we can enable things like

```
$$\href{https://github.com/shd101wyy}{github}$$
$$\htmlData{type=foo,id=bar}{x+y}$$
```
it currently fails (both in atom and vs code)
![image](https://user-images.githubusercontent.com/15909740/100639982-29f36680-3336-11eb-9d05-63eb757f96d1.png)

Thanks in advance for your consideration. Happy to help on this if pointed to the right place

**Comment by fredericcogny** [comment]
Edit. Through VS code I found the `~/mume/katex_config.js` file

however adding the below

```json
module.exports = {
  trust: true,
  throwOnError: false,
  strict: false
}
```

only helped with `herf` but not with `htmlData`

![image](https://user-images.githubusercontent.com/15909740/101340778-7aac1780-3880-11eb-98be-91775fceecce.png)

[exact same behaviour in atom]


But it works perfectly fine in vuepress with the below

```json
  markdown: {
    extendMarkdown: md => {
      md.use(require('markdown-it-texmath'), {
        engine: require('katex'),
        delimiters: 'dollars',
        katexOptions: {
          trust: true,
          throwOnError: false,
          strict: false
        }
      }
      )
    }
  },
```

![image](https://user-images.githubusercontent.com/15909740/101341138-ec846100-3880-11eb-9c47-b21dc7cbb4a2.png)

it might be an issue with mume?

@shd101wyy would you like me to move this issue upstream to https://github.com/shd101wyy/mume instead?

**Comment by Yue-Zhengyuan** [comment]
@fredericcogny It seems that this problem is now resolved (MPE version 0.6.2). 
<img width="1071" alt="Screen Shot 2022-03-10 at 17 32 13" src="https://user-images.githubusercontent.com/43436192/157632943-412bad20-f5f8-4d8a-ab68-94b31536ba49.png">

**Comment by fredericcogny** [comment]
nice one - confirmed it works - thanks

![image](https://user-images.githubusercontent.com/15909740/157720291-71021437-3efa-410e-9754-c1e61629f480.png)
